lib/oban/engines/lite.ex

```diff
@@ -146,8 +146,9 @@ defmodule Oban.Engines.Lite do
     select_query =
       queryable
       |> select([j], map(j, [:id, :queue, :state]))
-      |> where([j], j.state in ~w(completed cancelled discarded))
-      |> where([j], j.scheduled_at < ^time)
+      |> where([j], j.state == "completed" and j.completed_at < ^time)
```

Comment:
Chances are a `lite` database doesn't have nearly as many jobs, but we should still use `scheduled_at` for consistency.